apps/nsq_stat/nsq_stat.go

```diff
@@ -27,6 +27,8 @@ var (
 	channel          = flag.String("channel", "", "NSQ channel")
 	statusEvery      = flag.Duration("status-every", -1, "(deprecated) duration of time between polling/printing output")
 	interval         = flag.Duration("interval", 2*time.Second, "duration of time between polling/printing output")
+	httpConnectTimeout = flag.Duration("http-client-connect-timeout", 2*time.Second, "timeout for HTTP connect")
+	httpRequestTimeout = flag.Duration("http-client-request-timeout", 2*time.Second, "timeout for HTTP read/write (each)")
```

Comment:
The description should be `timeout for HTTP request`, it's not the read/write anymore after this set of changes.

(repeat for all locations of this description)